src/components/form-draft/testing.vue

```diff
@@ -21,6 +21,25 @@ except according to the terms contained in the LICENSE file.
             </enketo-fill>
           </template>
           <template #body>
+            <div v-if="form && form.entityRelated" class="panel-dialog">
```

Comment:
Just wondering, shouldn't we check for `entityRelated` on the draft rather than the form? Even if a form doesn't have a published version, I think we want to show this warning if the draft form definition would create entities.